database/migrations/2023_04_26_104633_create_landing_pages_tables_from_pages_table.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+class CreateLandingPagesTablesFromPagesTable extends Migration
+{
+    public function up()
+    {
+        // Create landing_pages tables
+
+        DB::statement('CREATE TABLE landing_pages AS TABLE pages');
+        DB::statement('ALTER TABLE landing_pages
+                ADD COLUMN visit_nav_buy_tix_label TEXT,
+                ADD COLUMN visit_nav_buy_tix_link TEXT,
+                ADD COLUMN visit_hours_intro TEXT,
+                ADD COLUMN visit_members_intro TEXT,
+                ADD COLUMN visit_admission_intro TEXT');
+        DB::statement('CREATE SEQUENCE landing_pages_id_seq AS integer');
+        DB::statement("ALTER TABLE landing_pages ALTER COLUMN id SET DEFAULT nextval('landing_pages_id_seq'::regclass)");
+        DB::statement('CREATE TABLE landing_page_types (
+            id INT NOT NULL PRIMARY KEY,
+            page_type VARCHAR(255) UNIQUE,
+            created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+            updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+        )');
+        DB::statement('CREATE SEQUENCE landing_page_types_id_seq AS integer');
+        DB::statement("ALTER TABLE landing_page_types ALTER COLUMN id SET DEFAULT nextval('landing_page_types_id_seq'::regclass)");
```

Comment:
All of this can be coded using the migration functions like so:

```
        Schema::create('landing_page_types', function (Blueprint $table) {
            $table->increments('id');
            $table->integer('page_type');
            $table->timestamps();
        });
```